table/block_based_table_reader.cc

```diff
@@ -115,6 +114,11 @@ void DeleteCachedEntry(const Slice& /*key*/, void* value) {
   delete entry;
 }
 
+void DeleteBlockContents(const Slice& /*key*/, void* value) {
```

Comment:
What was the problem with DeleteCachedEntry?

Reply:
Any thoughts on this?

Reply:
Hmm, Right I use DeleteCachedEntry